charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -45,13 +45,13 @@ groups:
   # 1. This helps evaluate if memory-based HPA should be re-introduced at 120%. And if it is re-opened what will be the impact on the avg number of replicas per active shoot in the seed clusters.
   # 2. This helps evaluate the benefit in reducing HPA scale down stabilization period (currently, 24h) to gain better resource utilization.
   - record: seed:apiserver_containers_over_cpu_scaling_threshold_total:count_by_container
-    expr: count(sum(rate(container_cpu_usage_seconds_total{container="kube-apiserver"}[5m])) by (namespace) / sum(kube_pod_container_resource_requests_cpu_cores{container="kube-apiserver"}) by (namespace) >= 0.8)
+    expr: count(sum(rate(container_cpu_usage_seconds_total{container="kube-apiserver"}[5m])) by (namespace) / sum(kube_pod_container_resource_requests{resource="cpu", unit="core", container="kube-apiserver"}) by (namespace) >= 0.8)
```

Comment:
In the same file there are still usages of `kube_pod_container_resource_requests_cpu_cores` - don't they need to be adapted as well?

https://github.com/gardener/gardener/blob/8594a502d2e5809841e06925cf075f9725def327/charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml#L22-L24

Reply:
oh, I'm not sure how that slipped through! Good catch and I'll update the PR.

Reply:
I verified the changes in the local setup, by checking the recording rules in the garden/prometheus.

Reply:
Thanks, appreciated!